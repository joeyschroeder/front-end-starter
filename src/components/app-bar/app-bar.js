import PropTypes from 'prop-types';
import React from 'react';
import styles from './app-bar.style.scss';

export const AppBar = (props) => {
  const { children } = props;

  return <div className={styles.root}>{children}</div>;
};

AppBar.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ]),
};

AppBar.defaultProps = {
  children: null,
};
