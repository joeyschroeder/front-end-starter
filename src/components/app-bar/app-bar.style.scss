@import "../../styles/config/variables";

.root {
  background: $color-white;
  border-bottom: $border-width solid $color-border;
}
