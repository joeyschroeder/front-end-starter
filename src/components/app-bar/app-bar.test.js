import { AppBar } from './app-bar';
import React from 'react';
import { create } from 'react-test-renderer';

describe('<AppBar />', () => {
  it('should render correctly', () => {
    const component = create(<AppBar />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
