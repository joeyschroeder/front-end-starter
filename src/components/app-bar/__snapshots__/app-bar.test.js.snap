// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render correctly 1`] = `
<div
  className="root"
/>
`;
